File: HELLO_ANDROID01/app/src/main/java/kr/co/aiai/app/MainActivity03.java
package kr.co.aiai.app;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity03 extends AppCompatActivity {
    Button btn;
    EditText et_dan;
    EditText et_disp;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main03);
        btn=findViewById(R.id.btn);
        et_dan=findViewById(R.id.et_dan);
        et_disp=findViewById(R.id.et_disp);
        btn.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view) {
               myclick();
            }
        });
    }
    void myclick(){
        int dan = Integer.parseInt(""+et_dan.getText());
        String txt="";
        for(int i=1;i<=9;i++){
            txt+=dan+"*"+i+"="+(dan*i)+"\n";
        }

        et_disp.setText(txt);
    }
}

File: HELLO_AI4/ANDROID/app/src/main/java/kr/co/aiai/myapp/MainActivity03.kt
package kr.co.aiai.myapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity03 : AppCompatActivity() {
    lateinit var tv:TextView
    lateinit var et_dan:EditText
    lateinit var et_disp:EditText

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main03)
        var btn = findViewById<Button>(R.id.btn)
        tv = findViewById<EditText>(R.id.et_dan)
        et_dan = findViewById<EditText>(R.id.et_dan)
        et_disp = findViewById<EditText>(R.id.et_disp)

        btn.setOnClickListener{
            myclick()
        }


    }
    fun myclick(){
        var dan =Integer.parseInt(tv.getText().toString());
        var txt=""
        var tdan=dan.toString();
        for (i in 1..9) {
            var result = dan*i;
            txt+=tdan+"*"+i.toString()+"="+result.toString()+"\n";
        }
        et_disp.setText(txt);
    }

}